### DISCUSSION TITLE
Is session resumed and paused automatically?

### DISCUSSION THREAD

**Discussion by sarimmehdi** [sceneview/sceneform-android#298] [Q&A] [answered]
I was looking at this [sample](https://github.com/SceneView/sceneform-android/blob/master/samples/ar-model-viewer-java/src/main/java/com/google/ar/sceneform/samples/gltf/MainActivity.java) and I noticed that there is no onPause, onResume, or onDestroy method being overridden. In the original Sceneform, if I recall correctly, we would have to resume and pause the session automatically.

**Comment by grassydragon** [comment, marked as answer]
Yes, it is paused and resumed automatically in the `BaseArFragment`: https://github.com/SceneView/sceneform-android/blob/master/ux/src/main/java/com/google/ar/sceneform/ux/BaseArFragment.java#L245
